code-challenges.test.js

```diff
@@ -33,21 +74,65 @@ const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
-
+//pseudocode:
+// Create a function named evenlyDivByThree
+//input: object 1, object 2, object 3
+//output: whether number value in objects are evenly divisible by three, meaning no remainder.
+//will need 3 return statements
+// conditionals to do the math and compare to input values.
+//return string that states if it is evenly divisible by three.
+const evenlyDivByThree = () => {
+    if (object1 === 0) {
+        return "15 is divisible by three"
+    } else if (object2 === 0) {
+        return "0 is divisible by three"
+    } else if (object3 === 0){
+        return "-7 is not divisible by three"
+    } else {
+        return "error"
+    }
+}
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+    describe("allCaps", () => {
+        it("takes in an array of words and returns an array with all the first letters of each words capitalized.", () => {
+           expect (toCaps.toEqual(randomNouns1))
+        })
+    })
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
+//Pseudocode:
+//input: An array
+//output: Returns same length array, but the first letter of each word is capatalized.
+//will need to use a built in method that will work on an array.
```

Comment:
What are you currently getting back when you run your tests?  If your tests are failing, you should be adding the error message and write about the steps you took to problem-solve this.  The pseudocode provided is not going to provide much help go get you to the next step.  Make sure to spend some time writing about the process, otherwise, this will be marked as complete.